### DISCUSSION TITLE
Liquidsoap conf editor field

### DISCUSSION THREAD

**Discussion by rmacari** [AzuraCast/AzuraCast#5886] [General]
Hello,

Can I ask if the field to edit the liquidsoap conf was removed? 
I've tried to manually edit it, but it's overwritten in every restart. 

Thanks.

**Comment by BusterNeece** [comment]
It's not a single field any more, it's a full page called "Edit Liquidsoap Configuration" under the "Broadcasting" sidebar menu, and it certainly still exists.

**Reply by rmacari** [reply]
Thanks! I was looking at the old place and could not find it there. Appreciate the help. :)
